### ISSUE TITLE
[BUG]: Contents loads slowly

### ISSUE THREAD

**Issue by barathrm** [martinrotter/rssguard#854]
### Brief description of the issue

Apologies if this is already covered somewhere, a search didn't turn up anything relevant.

Basically, loading content when clicking on an entry is slow. It feels like the internal browser starts (down-) loading the entry only once clicking the entry and that it's not precached. I couldn't find any relevant setting for enabling that. I'm coming from akregator where there's no delay at all from clicking on an entry to it being displayed in the internal browser.

Attached are two screen recordings showing the difference between akregator and rssguard.

https://user-images.githubusercontent.com/76487253/206406155-7fa075f2-6dc8-47a2-a201-de89875fc2fe.mp4

https://user-images.githubusercontent.com/76487253/206406160-16e2f384-6cf7-4ac2-97aa-35e23d35f458.mp4

### How to reproduce the bug?

1. click an RSS entry
2. notice the time it takes to load the content

### What was the expected result?

Content should be displayed instantaneously.

### What actually happened?

Content takes several seconds to load, depending on the website/feed.

### Debug log

-

### Operating system and version

* OS: linux, openSUSE Tumbleweed
* RSS Guard version: 4.2.3

**Comment by martinrotter** [comment]
Yes, run "rssguard" in terminal and copy-paste all terminal output it creates (when triggering the buggy behavior).

**Comment by martinrotter** [comment]
I OVERLOOKED.

**Comment by martinrotter** [comment]
Can you install "nowebengine" variant of RSS Guard and check if there is any difference regarding the speed of displaying of articles? Or just stay with your rssguard and run it with "-w" parameter.

Post video pls.

**Comment by barathrm** [comment]
Running rssguard with `-w` worked, loading entries is now instantaneous! Obviously the formatting is different, but that's fine for me. I guess how fast the web-engine is isn't necessarily up to you. Thanks for the help :) 

Attached log and video.




https://user-images.githubusercontent.com/76487253/206421717-f1ba1a3a-2484-4449-a79e-f03a9e0940ba.mp4

[rssguard2.log](https://github.com/martinrotter/rssguard/files/10184390/rssguard2.log)

**Comment by martinrotter** [comment]
That is clearly some problem with WebEngine component.

1, Do you have AdBlock enabled?
2. How did you install RSS Guard?

**Comment by barathrm** [comment]
1. I never enabled it after installing and couldn't find it in the rssguard settings itself. Looking a bit more  "Web browser & Tabs/Adblock/Show adblock settings" shows it as disabled.
2. Installed it from the openSUSE Tumbleweed repos. The package is maintained at https://build.opensuse.org/package/show/X11:Utilities/rssguard. It looks like it's built the standard way, except for making sure that the library follows name versioning conventions.

In case it matters, I currently have libqt5-qtwebengine version 5.15.11-1.4 installed.

In case it matters, the spec uses the following to build rssguard
```
%cmake
%cmake_build
``` 

Which should evaluate to
```
$ rpm --eval %{cmake}

    find . -type f -name CMakeLists.txt \
        -exec sed -i -re '/^[[:blank:]]*[sS][eE][tT][[:blank:]]*\([[:blank:]]*(CMAKE_BUILD_TYPE|CMAKE_COLOR_MAKEFILE|CMAKE_INSTALL_PREFIX|CMAKE_VERBOSE_MAKEFILE).*\)/{s/^/#IGNORE /}' {} + 
    mkdir -p build 
    cd build 
    /usr/bin/cmake $OLDPWD/. \
%if "/usr/bin/make" == "/usr/bin/make" 
        -G"Unix Makefiles" \
%else 
        -GNinja \
%endif 
        -DCMAKE_INSTALL_PREFIX:PATH=/usr \
        -DINCLUDE_INSTALL_DIR:PATH=/usr/include \
        -DLIB_INSTALL_DIR:PATH=/usr/lib64 \
        -DSYSCONF_INSTALL_DIR:PATH=/etc \
        -DSHARE_INSTALL_PREFIX:PATH=/usr/share \
        -DCMAKE_INSTALL_LIBDIR:PATH=lib64 \
        -DCMAKE_INSTALL_LIBEXECDIR=/usr/libexec \
        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
        -DCMAKE_C_FLAGS="${CFLAGS:--O2 -g -m64 -fmessage-length=0 -D_FORTIFY_SOURCE=2 -fstack-protector -funwind-tables -fasynchronous-unwind-tables}" \
        -DCMAKE_CXX_FLAGS="${CXXFLAGS:--O2 -g -m64 -fmessage-length=0 -D_FORTIFY_SOURCE=2 -fstack-protector -funwind-tables -fasynchronous-unwind-tables}" \
        -DCMAKE_Fortran_FLAGS="${FFLAGS:--O2 -g -m64 -fmessage-length=0 -D_FORTIFY_SOURCE=2 -fstack-protector -funwind-tables -fasynchronous-unwind-tables}" \
        -DCMAKE_EXE_LINKER_FLAGS=" -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now" \
        -DCMAKE_MODULE_LINKER_FLAGS=" -Wl,--as-needed" \
        -DCMAKE_SHARED_LINKER_FLAGS=" -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now" \
%if "lib64" == "lib64" 
        -DLIB_SUFFIX=64 \
%endif 
%if 1599 <= 1500 
        -DCMAKE_SKIP_INSTALL_RPATH:BOOL=ON \
%endif 
        -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
        -DBUILD_SHARED_LIBS:BOOL=ON \
        -DBUILD_STATIC_LIBS:BOOL=OFF \
        -DCMAKE_COLOR_MAKEFILE:BOOL=OFF \
        -DCMAKE_INSTALL_DO_STRIP:BOOL=OFF \
        -DCMAKE_MODULES_INSTALL_DIR=/usr/lib64/cmake/%{name}

$ rpm --eval %{cmake_build}
/usr/bin/make \
%if "/usr/bin/make" == "/usr/bin/make" 
    -O VERBOSE=1 \
%else 
    -v \
%endif 
```

**Comment by martinrotter** [comment]
Hard to say. Since no-webengine variant is good enough for you, I would close this. If anyone else comes with same problem, we will somehow deal with it.
